cli/stork-publisher-agent/model.go

```diff
@@ -0,0 +1,118 @@
+package stork_publisher_agent
+
+import "math/big"
+
+type MessageType string
+
+const WildcardSubscriptionAsset = "*"
+
+type ConnectionId string
+type AuthToken string
+
+type WebsocketMessage[T any] struct {
+	connId  ConnectionId
+	Type    string `json:"type"`
+	Error   string `json:"error,omitempty"`
+	TraceId string `json:"trace_id,omitempty"`
+	Data    T      `json:"data,omitempty"`
+}
+
+type ErrorMessage struct {
+	Error string `json:"error"`
+}
+
+type (
+	OracleId          string
+	AssetId           string
+	EvmPublisherKey   string
+	EvmPrivateKey     string
+	StarkPublisherKey string
+	StarkPrivateKey   string
+	PublisherKey      string
+	SignatureType     string
+	QuantizedPrice    string
+)
+
+const EvmSignatureType = SignatureType("evm")
+const StarkSignatureType = SignatureType("stark")
+
+// Incoming
+type PriceUpdatePullWebsocket struct {
```

Comment:
Using different input objects for the pull-based websocket and the push based websocket. The data we're currently pulling has a `p` field so it felt weird to call that field Value.

It also seems like we can better avoid precision loss by converting strings to big.Float than by converting floats to big.Float, so for the push based message I'm using a string type rather than float.